gomod-sync/config.sample.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I made this file inspired by other pieces of software I used before that provided a file like this. But this might not be necessary in our case since we have the README, and I understand how it can be confusing.

The file is now removed.